pkg/option/config.go

```diff
@@ -632,6 +635,10 @@ const (
 	// entries.
 	SockRevNatEntriesName = "bpf-sock-rev-map-max"
 
+	// EgressPolicyMapName configures max entries for egress gateway's policy
+	// map.
+	EgressPolicyMapName = "egress_gw_policy_max"
```

Comment:
```suggestion
	EgressPolicyMapName = "egress-gw-policy-max"
```

Consistency with the other option names.

Reply:
Thanks!